### ISSUE TITLE
Automate creating Firebase proeject

### ISSUE THREAD

**Issue by gurrpi** [quadratic-funding/mpc-phase2-suite#199]
[Set up and manage a Firebase project using the Management REST API](https://firebase.google.com/docs/projects/api/workflow_set-up-and-manage-project)

**Comment by 0xjei** [comment]
UP: Do we still need this workflow? @gurrpi

**Comment by gurrpi** [comment]
> UP: Do we still need this workflow? @gurrpi

It would be more efficient to have automated provisioning rather than having users do it manually.

**Comment by 0xjei** [comment]
> > UP: Do we still need this workflow? @gurrpi
> 
> It would be more efficient to have automated provisioning rather than having users do it manually.

Agree. I'm assuming that we could work on this after the first release as it's a non-blocking issue.
